internal/common/actions/run_js/action.bzl

```diff
@@ -0,0 +1,43 @@
+load("//internal/npm_packages:rule.bzl", "NpmPackagesInfo")
+
+def run_js(
```

Comment:
Thanks for this. Rules that use it look a lot better.

Reply:
I agree, I think that I have it about where I want it. I was trying to abstract over some of the Bazel specific stuff with the combination of this and the `BazelAction` js function. Currently the primary thing that it handles is parsing a param file if bazel decides to generate one. But would be nice in the future to have it expose the ability to be run as a bazel worker. But I haven't  looked too deeply into that yet.